R/forecast_time_series.R

```diff
@@ -453,6 +453,7 @@ forecast_time_series <- function(input_data,
                                                  .packages = get_export_packages(), 
                                                  .export = c("fcst_prep", "get_cores"),
                                                  .options.azure = list(maxTaskRetryCount = 0, autoDeleteJob = TRUE, 
+                                                                       timeout = 60 * 60 * 24 * 7, # timeout after a week
```

Comment:
Global config.

Reply:
Hey Aadharsh, I'm hesitant to add more function arguments to the `forecast_time_series` function, since it's already quite large and we're trying to move away from Azure Batch. If we add a new function argument like "azure_batch_timeout" then it will most likely get deprecated in the next major release when we roll out things like spark integration. Not sure if users would want to expand this timeout farther than a week and would most likely not want to decrease the timeout to something like 12 hours. What do you think? 

We can always increase the timeout if users reach out with feedback.

Reply:
It does beg the question if we should institute a more general timeout feature that can be applied in Azure Batch or when running on a users local machine. That would open the door up for timeout functionality in new parallel compute options like spark.

Reply:
Yeah, you have me convinced. These are just two places including one of them legacy.